src/api/SFA.DAS.Provider.Events.Api/Web.config

```diff
@@ -18,9 +18,9 @@
   <appSettings>
     <add key="LogLevel" value="Debug" />
     <add key="EnvironmentName" value="LOCAL" />
-    <add key="LoggingConnectionString" value="server=.;database=DasProviderEventsApi;trusted_connection=true;" />
-    <add key="MonthEndConnectionString" value="server=.;database=DasProviderEventsApi;trusted_connection=true;" />
-    <add key="EventsConnectionString" value="server=.;database=DasProviderEventsApi;trusted_connection=true;" />
+    <add key="LoggingConnectionString" value="server=.;database=ProviderEventsAT;trusted_connection=true;" />

```

Comment:
You will want to revert these changes otherwise it'll stop the API running for everything else (unless they rename their databases).